### PR TITLE
feature: Integrate Sign up with Airtable form

### PR DESCRIPTION
**Pull request by GarfDev**
Closing #51 

- Add a airtable integrate at signupFor method (after user confirmed they email)
- Add airtableMigrate command that migrates all user's data from the Account model to database

### CODE REVIEW

**Review by peterpolman — APPROVED** [review]

**Comment by peterpolman** [comment]
@GarfDev Looking good! I think we need to enable Airtable calls for prod only. Probably add a env bar similar to API project to this project.

File: .env.ci [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Docker MongoDB URI
 MONGODB_URI=mongodb://root:root@auth_mongo:27017/auth?authSource=admin&ssl=false
 MONGODB_URI_TEST_OVERRIDE=mongodb://root:root@auth_mongo:27017/auth_test?authSource=admin&ssl=false
+AIRTABLE_API_KEY=xxxxxxxxxxxxxxxxx
```
Comment (bram-rongen) [inline .env.ci:4]: You don't need to replicate vars from `.env.example` in `.env.ci`. The `.env.ci` file is loaded in addition to the `.env.example` file during CI runs.

**Comment by bram-rongen** [comment]
> @GarfDev Looking good! I think we need to enable Airtable calls for prod only. Probably add a env bar similar to API project to this project.

Personally I'd try to do this on the basis of the credentials being set. When they are not set you don't make any calls. You could log a message when credentials are unset on initialisation so its easy to figure out why calls are not being triggered.. But having a separate env var adds a var without value..

**Comment by peterpolman** [comment]
Even better 👌

**Review by peterpolman — APPROVED** [review]

**Pull request merged**